### DISCUSSION TITLE
Github Developer Student Pack Error

### DISCUSSION THREAD

**Discussion by Fahad12405** [community/community#111399] [GitHub Education]
### Hi everyone,

I have completed all the required steps as requested, including providing a valid ID card and filling out the billing information. However, I encountered an issue where the application was deemed unsuccessful, with the message stating: 'Your application cannot be reviewed until you fix the following: Your uploaded proof has already been used previously.' Could you please assist me in resolving this matter so that I may successfully apply for the GitHub Education pack?

**Comment by ngljcb** [comment]
It seems like your application for the GitHub Education Pack was rejected because the proof of eligibility you provided has already been used previously. To resolve this, try using a different document as proof of eligibility, like a **_certificate of enrollment_** which is typically accepted as proof of eligibility for the GitHub Education Pack. Just ensure that the certificate clearly indicates your **current enrollment status** at an accredited educational institution, including the **relevant dates**. 

Make sure to follow the guidelines provided by GitHub for acceptable proof of eligibility.

**Comment by probablynotnmp** [comment]
I recommend reaching out to GitHub support directly. They should be able to assist you further in addressing this specific issue and providing guidance on how to proceed :)
